pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/ecdh_aead_private_key_manager.go

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/tink/go/core/registry"
+	"github.com/google/tink/go/keyset"
+	"github.com/google/tink/go/subtle/hybrid"
+	tinkpb "github.com/google/tink/proto/tink_go_proto"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/jwe/subtle/ecdhes"
+	ecdhespb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/ecdhes_aead_go_proto"
+)
+
+const (
+	ecdhesPrivateKeyVersion = 0
+	ecdhesPrivateKeyTypeURL = "type.googleapis.com/google.crypto.tink.EcdhesAeadPrivateKey"
```

Comment:
Is this correct for a type defined outside of the tink project?

Reply:
Should I change to `hyperledger.aries.com`? This identifies the new key types introduced in this change.

Reply:
updated to `type.hyperledger.org/hyperledger.aries.crypto.tink.EcdhesAeadPrivateKey`